src/wallet.py

```diff
@@ -0,0 +1,92 @@
+import json
+from json.decoder import JSONDecodeError
+
+import structlog
+from nacl.encoding import HexEncoder
+from nacl.signing import SigningKey
+import base64
+from cryptography.fernet import Fernet
+
+
+logger = structlog.get_logger(__name__)
+
+class wallet:
```

Comment:
Can we name it `Wallet`? To follow the [PEP8](https://www.python.org/dev/peps/pep-0008/#class-names) convection?